gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStoragePathTest.java

```diff
@@ -170,38 +169,38 @@ public void testNormalize() {
   }
 
   @Test
-  public void testNormalize_dot_becomesBlank() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
+  public void testNormalize_dot_becomesBlank() throws IOException {
+    try (CloudStorageFileSystem fs = helper.forBucket("doodle")) {
       assertThat(fs.getPath("").normalize().toString()).isEqualTo("");
       assertThat(fs.getPath(".").normalize().toString()).isEqualTo("");
     }
   }
 
   @Test
-  public void testNormalize_trailingSlash_isPreserved() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
+  public void testNormalize_trailingSlash_isPreserved() throws IOException {
+    try (CloudStorageFileSystem fs = helper.forBucket("doodle")) {
       assertThat(fs.getPath("o/").normalize().toString()).isEqualTo("o/");
     }
   }
 
   @Test
-  public void testNormalize_doubleDot_becomesBlank() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
+  public void testNormalize_doubleDot_becomesBlank() throws IOException {
+    try (CloudStorageFileSystem fs = helper.forBucket("doodle")) {
       assertThat(fs.getPath("..").normalize().toString()).isEqualTo("");
       assertThat(fs.getPath("../..").normalize().toString()).isEqualTo("");
     }
   }
 
   @Test
-  public void testNormalize_extraSlashes_getRemoved() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/mziccard/gcloud-java/file/2326131133/issues/source?bid=3123107&fileBranchId=3190440#l195)